cli/tests/pcluster/validators/test_all_validators.py

```diff
@@ -247,9 +247,11 @@ def test_validators_are_called_with_correct_argument(test_datadir, mocker):
         [call(kms_key_id="1234abcd-12ab-34cd-56ef-1234567890ab", encrypted=True)]
     )
     fsx_architecture_os_validator.assert_has_calls([call(architecture="x86_64", os="alinux2")])
-    duplicate_mount_dir_validator.assert_has_calls(
-        [call(mount_dir_list=["/my/mount/point1", "/my/mount/point2", "/my/mount/point3", "/scratch"])]
-    )
+    mount_dir_list = duplicate_mount_dir_validator.call_args.kwargs["mount_dir_list"]
+    expected_mount_dir_list = ["/my/mount/point1", "/my/mount/point2", "/my/mount/point3", "/scratch", "/scratch_head"]
+    assert_that(mount_dir_list).is_length(len(expected_mount_dir_list))
```

Comment:
Since the ephemeral mount dirs are from a set. The order of set is not consistent. So we need to check the items in the list one by one